lib/pipedrive/utils.rb

```diff
@@ -10,7 +10,13 @@ def follow_pagination(method, args, params, &block)
         res = __send__(method, *args, params.merge(start: start))
         break if !res.try(:data) || !res.success?
 
-        res.data.each(&block)
+        if res.data.try(:items).present?
+          res.data.items.each do |item|
+            yield(item.item)
```

Comment:
stylistic suggestion, since `yield` is a keyword:
```suggestion
            yield item.item
```